packages/m/moltenvk/xmake.lua

```diff
@@ -12,6 +12,9 @@ package("moltenvk")
     add_versions("v1.1.4", "f9bba6d3bf3648e7685c247cb6d126d62508af614bc549cedd5859a7da64967e")
     add_versions("v1.1.0", "0538fa1c23ddae495c7f82ccd0db90790a90b7017a258ca7575fbae8021f3058")
 
+    add_links("MoltenVKShaderConverter", "MoltenVK")
+    add_frameworks("CoreGraphics")
```

Comment:
move it to `if is_plat("macosx")` branch

Reply:
but this package only supports macosx and iphoneos plat

Reply:
Keeping a consistent configuration style for all packages allows users to read the configuration more clearly. Also, not everyone knows that this package only supports macosx, which can lead to users incorrectly assuming that add_frameworks can be used on all platforms.